### DISCUSSION TITLE
Unable to stop spot instances

### DISCUSSION THREAD

**Discussion by tomhall-inst** [npalm/terraform-aws-gitlab-runner#1049] [Q&A]
I am seeing the following error with spot instances -  {
    "level": "error",
    "msg": "UnsupportedOperation: You can't stop the Spot Instance 'i-128ee635e7e0476be' because it is associated with a one-time Spot Instance request. You can only stop Spot Instances associated with persistent Spot Instance requests.",
    "name": "runner-y-ese1xb-lt-ci-runners-v2-1700497615-f056900d",
    "operation": "stop",
    "time": "2023-11-20T16:31:16Z"
}. Anyone seen this?  Wondering if I configured something incorrectly.

**Comment by kayman-mk** [comment]
I have seen this somewhere, but can't remember where. All my spot instances are correctly terminated. I haven't seen any dangling instances so far.
